inc/class-crazylazy.php

```diff
@@ -103,7 +103,7 @@ public function load_plugin_textdomain() {
 	 */
 	public static function prepare_images( $content ) {
 		/* No lazy images? */
-		if ( strpos( $content, '-image' ) === false && strpos( $content, 'avatar-' ) === false  ) {
+		if ( strpos( $content, '-image' ) === false && strpos( $content, 'avatar-' ) === false  && strpos( $content, 'attachment-' ) === false  ) {
```

Comment:
It looks like there are two double spaces here. Am I right or is this a display issue?

Reply:
You are right. Thanks for catching it. I still have to setup my new Linux dev environment with my quality tools ;)